lambda/main.tf

```diff
@@ -0,0 +1,64 @@
+data "aws_caller_identity" "current" {}
```

Comment:
Would be nice to include a short README in the `lamba/` folder explaining the purpose of the module. Looks like the primary benefit is that it automatically creates the right role for your lambda to be able to run and write logs, right?